### ISSUE TITLE
[Question/Feature Request] Difference? Chart Support

### ISSUE THREAD

**Issue by Scottlr** [apexcharts/apexcharts.js#564]
Hey, was curious if you can generate charts similar to this using Apex Charts. If not, is this a chart you would consider supporting?

![image](https://user-images.githubusercontent.com/7180731/57018146-26f43980-6c1a-11e9-891d-c2b44651ddbd.png)

![image](https://user-images.githubusercontent.com/7180731/57018160-31163800-6c1a-11e9-8e80-4bbe0952635a.png)

I couldn't seem to find an example, within your demo pages.

I'm not sure of the exact name for this chart, hence the uncertainty in the issue title :smile:

**Comment by junedchhipa** [comment]
Yes, Range Area Charts are being considered and will be implemented soon.
Thanks for the suggestion.

**Comment by junedchhipa** [comment]
Duplicate of #184

**Comment by Scottlr** [comment]
Thanks for the response! I think there's a slight difference to this graph and range area graph in that that I'd want to have a different fill colour when "min value exceeds the max value" in range terms.

For instance, take the second graph I posted. The red line represents the money I've spent, and the green is the money I've earnt.

When that green line is above the red line, I want that fill area to be green (spent less than I've earned). When the red line is above the green, fill that area with red (spent more than I've earned margin).
